internal/pkg/template/templates/environment/partials/custom-resources.yml

```diff
@@ -35,7 +35,7 @@ HTTPSCert:
 CustomDomainAction:
   Metadata:
     'aws:copilot:description': 'Add an A-record to the hosted zone for the domain alias'
-  Condition: HasAliases
+  Condition: ManagedAliases
```

Comment:
is it possible that we still want this action if one LBWS has no `alias` but another LBWS has an `alias`?

Reply:
then `Aliases` would be non-empty and `HasAliases` would render true, as long as the dns is delegated (i.e. `copilot app init --domain` was executed